core/base/projectionFromTable/ProjectionFromTable.h

```diff
@@ -0,0 +1,132 @@
+/// \ingroup base
+/// \class ttk::ProjectionFromTable
+/// \author Mathieu Pont <mathieu.pont@lip6.fr>
+/// \date 2022.
+///
+/// This module defines the %ProjectionFromTable class that
+/// projects on a surface points in a vtkTable.
+///
+
+#pragma once
+
+// ttk common includes
+#include <Debug.h>
+#include <Geometry.h>
+#include <Triangulation.h>
+
+namespace ttk {
+
+  /**
+   * The ProjectionFromTable class provides methods that
+   * projects on a surface points in a vtkTable.
+   */
+  class ProjectionFromTable : virtual public Debug {
+
+  protected:
+  public:
+    ProjectionFromTable();
+
+    int preconditionTriangulation(AbstractTriangulation *triangulation) {
```

Comment:
This method seems to be quite useless...

Reply:
Removed in the next commit!